test/src/input.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { Input } from "@moai/core";
+import { useState } from "react";
+
+const InputTesting = () => {
+	const [value, setValue] = useState("");
+	return (
+		<div>
+			<Input value={value} setValue={setValue} data-testid="input" />
+			<div data-testid="div">{value}</div>
+			<button data-testid="button" onClick={() => setValue("Goodbye")}>
+				Change
+			</button>
+		</div>
+	);
+};
+
+describe("Testing Input Component", () => {
+	test("div value should change after Input value change", async () => {
+		const spy = jest.spyOn(console, "error");
+		spy.mockImplementation(() => void 0);
```

Comment:
Why do we need to mute the error?
- If we do need to mute the error, please put the explanation (e.g. what's the error) as a comment here
- However, maybe this is unnecessary?